MAD.Integration.TableauCRM/Jobs/JobManager.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Hangfire;
+using MAD.Integration.TableauCRM.Data;
+
+namespace MAD.Integration.TableauCRM.Jobs
+{
+    public class JobManager
+    {
+        private readonly ConfigurationDbContext dbContext;        
+        private readonly IRecurringJobManager recurringJobManager;
+        private readonly IBackgroundJobClient backgroundJobClient;
+
+        public JobManager(ConfigurationDbContext dbContext, IRecurringJobManager recurringJobManager, IBackgroundJobClient backgroundJobClient)
```

Comment:
Make sure to use the IRecurringJobFactory otherwise the CRON override hooks in settings.json won't work

Reply:
Doneskies. Wanna add the RemoveIfExists method and any other useful functions to IRecurringJobFactory in the future? Then we don't need IRecurringJobManager.

Reply:
Do eet. Can you also have a look into how possible it is to implement our own version IRecurringJobManager so we don't have two? Our own implementation should just add the additions of IReccurringJobFactory but under the interface of IReccuringJobManager. It should just act as a proxy for the normal IRecurringJobManager with the extra hooks from IRecurringJobFactory